net/table_net_certificate.go

```diff
@@ -142,16 +143,27 @@ func tableNetCertificateList(ctx context.Context, d *plugin.QueryData, h *plugin
 		InsecureSkipVerify: true,
 	}
 
+	tcpConnectionCreated := false
 	addr := net.JoinHostPort(dn, "443")
 	dialer := &net.Dialer{
 		Timeout: time.Duration(3) * time.Second, // short, certificates should be fast
+		Control: func(network, address string, c syscall.RawConn) error { // This gets called once the TCP connection gets opened before handshake
+			tcpConnectionCreated = true
+			return nil
+		},
 	}
 
 	conn, err := tls.DialWithDialer(dialer, "tcp", addr, &cfg)
 	if err != nil {
+		if tcpConnectionCreated {
+			plugin.Logger(ctx).Error("net_certificate.tableNetCertificateList", "failed to perform TLS handshake: ", err)
```

Comment:
```suggestion
			plugin.Logger(ctx).Error("net_certificate.tableNetCertificateList", "failed to perform TLS handshake:", err)
```

do we need the extra space, or does the plugin logger handle this?